### PR TITLE
feature/auth

### PR DESCRIPTION
**Pull request by turchin-aa**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  app.use(express.json())
+
+  const port = process.env.PORT || 8000
+
+  app.use('/api/auth', router)
+
+  dotenv.config()
+
+  const db = new Sequelize('devico_finalproject', 'root', '123456', {
```
Comment (VolodymyrDevico) [inline index.ts:16]: Move to .env

File: index.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    host: 'localhost',
+    dialect: 'mariadb'
+  })
+
+  const start = async () => {
+    try {
+      await db.authenticate()
+      console.log('Соединение с БД было успешно установлено')
```
Comment (VolodymyrDevico) [inline index.ts:24]: On english pls

File: index.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      
+    } catch (e) {
+      console.log('Невозможно выполнить подключение к БД: ', e)
+      process.exit(1)
+    } 
+  }
+
+  start()
+
+  const User = db.define("user", {
```
Comment (VolodymyrDevico) [inline index.ts:37]: move to another folder

File: routes/auth.routes.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
+import { Router, Request, Response } from "express"
+import bcrypt from "bcryptjs"
+import jwt from "jsonwebtoken"
+import User from '../index.js'
+
+const router = Router()
+
+router.post('/register', async (req : Request, res : Response) => {
+    try {
+        const {email, password} = req.body
```
Comment (VolodymyrDevico) [inline routes/auth.routes.ts:10]: Fix prettier

**Review by VolodymyrDevico — CHANGES_REQUESTED** [review]

**Review by ntnshpig — COMMENTED** [review]
Nice job, but sometimes it is hard to read.

**Pull request closed without merging**